src/modules/drawing/DrawingModule.vue

```diff
@@ -147,37 +150,34 @@ export default {
             }
             return null
         },
-        isNewDrawing() {
-            return !this.kmlIds
-        },
         isDrawingModified() {
             return this.savingStatus !== SavingStatus.INITIAL
         },
     },
     watch: {
         async show(show) {
-            /**
-             * Makes it possible to abort the toggle on error by setting
-             * "this.abortedToggleOverlay=true" before calling "this.toggleDrawingOverlay()", as
-             * calling "this.toggleDrawingOverlay()" will retrigger this function.
-             */
-            if (this.abortedToggleOverlay) {
-                this.abortedToggleOverlay = false
-                return
-            }
```

Comment:
Removing this protection while retrying the saving automatically when the prior saving failed (line 316) is very dangerous. Because now if you loose the internet connection and quit the drawing, not only will you loose your changes, but in the background the drawing module will still continuousely try to save the drawing. So when the internet connection comes again, as the drawing layer is empty when not in drawing mode, it will overwrite your kml drawing with a blank one.
So I would either improve the saving retry policy so that it stops to try to save the drawing when quitting the drawing mode, or readd the above protection, or both. Or/and clear the kml metadata when quitting drawing mode to prevent any accidental overwriting of the kml.

Reply:
I removed this protection as it did not work correctly. It simply prevented to close the menu without retry of the failed saved and during my test (I changed the url of the backend to force a failed saving) the error message was not displayed. So that was not very user friendly as you could not do the action that you want without any indication why. I did not debug to see why the saving status of error was not displayed though. 
I think we should not try to make decision for the user, this will not be understood. Automatic retry on auto save is the best IMHO.
However you brought a very good point with the auto retry continuing when the drawing menu is closed ! I'll fix this, see #331